arm-oraclelinux-wls-dynamic-cluster/src/main/arm/createUiDefinition.json

```diff
@@ -681,11 +681,37 @@
                                     "required": true
                                 }
                             },
+                            {
+                                "name": "keyVaultPrivateKeyAliasSecretName",
+                                "type": "Microsoft.Common.TextBox",
+                                "visible": "true",
+                                "label": "The name of the secret in the specified Key Vault whose value is the private key Alias",
+                                "defaultValue": "",
+                                "toolTip": "Use only letters and numbers",
+                                "constraints": {
+                                    "required": true,
+                                    "regex": "^[a-z0-9A-Z]{1,30}$",
+                                    "validationMessage": "The value must be 1-30 characters long and must only contain letters and numbers."
+                                }
+                            },
+                            {
+                                "name": "keyVaultPrivateKeyPassPhraseSecretName",
+                                "type": "Microsoft.Common.TextBox",
+                                "visible": "true",
+                                "label": "The name of the secret in the specified Key Vault whose value is the passphrase for the private key",
+                                "defaultValue": "",
+                                "toolTip": "Use only letters and numbers",
+                                "constraints": {
+                                    "required": true,
+                                    "regex": "^.{6,}$",

```

Comment:
@edburns Hi Ed, I think this is the textbox for entering the **Secret Name of a passphrase**, not the passphrase itself. So the regex and message here should be the same as above.
"regex": "^[a-z0-9A-Z]{1,30}$",
"validationMessage": "The value must be 1-30 characters long and must only contain letters and numbers."